tool/tctl/common/auth_command.go

```diff
@@ -394,6 +410,23 @@ func (a *AuthCommand) ListAuthServers(ctx context.Context, clusterAPI auth.Clien
 	return nil
 }
 
+// GenerateCRLForCA generates a certificate revocation list for a certificate
+// authoritity.
+func (a *AuthCommand) GenerateCRLForCA(ctx context.Context, clusterAPI auth.ClientI) error {
+	certType := types.CertAuthType(a.caType)
+	if err := certType.Check(); err != nil {
+		return trace.Wrap(err)
+	}
+
+	crl, err := clusterAPI.GenerateCertAuthorityCRL(ctx, certType)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	fmt.Println(string(crl))
```

Comment:
Does this write binary output to the terminal?

Reply:
Yes. It is the same as how we're printing the contents on [`auth export`](https://github.com/gravitational/teleport/blob/master/lib/client/ca_export.go#L268).

Also, I've verified the contents with `openssl crl -in db.crl -inform DER -text -noout`:
```
Certificate Revocation List (CRL):
        Version 2 (0x1)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: /O=root.teleport.dev/CN=root.teleport.dev/serialNumber=000000000000000000000000000000000000000
        Last Update: Mar  3 00:00:00 2023 GMT
        Next Update: Mar  2 00:00:00 2024 GMT
...
```

Reply:
Couldn't that have side effects, like including control characters that change the color of the terminal or put it in raw mode?

Reply:
I could. As I can see, we're usually using the command on our docs to direct the output to a file (e.g.: [`tctl auth export --type=windows > user-ca.der`](https://goteleport.com/docs/desktop-access/active-directory-manual/#export-the-teleport-ca)). Do you think it is better to have a `--out/-o` as we do on `tctl auth sign`? In that case, users could still output to stdout and direct it to a file.